### ISSUE TITLE
False positive "no-unused-import": user-defined value type defined as constant

### ISSUE THREAD

**Issue by PaulRBerg** [solhint-community/solhint-community#19]
In the UD60x18 value type[^1] in my PRBMath library, there is a constant called `ZERO`:

https://github.com/PaulRBerg/prb-math/blob/28055f6cd9a2367f9ad7ab6c8e01c9ac8e9acc61/src/ud60x18/Constants.sol#L51

Importing and using this constant leads to an "imported name ZERO is not used" error:

```solidity
pragma solidity >=0.8.19 <0.9.0;

import { UD60x18, ZERO } from "@prb/math/UD60x18.sol";

contract SetProtocolFee_Integration_Fuzz_Test {
    function testFuzz_SetProtocolFee(UD60x18 newProtocolFee) external {
        newProtocolFee = _bound(newProtocolFee, 1, MAX_FEE);
        vm.expectEmit({ emitter: address(comptroller) });
        emit SetProtocolFee({ admin: users.admin, asset: dai, oldProtocolFee: ZERO, newProtocolFee: newProtocolFee });
    }
}
```

[^1]: https://blog.soliditylang.org/2021/09/27/user-defined-value-types

**Comment by fvictorio** [comment]
The other two issues make complete sense, but I think I'm missing something here. How is `ZERO` used?

**Comment by PaulRBerg** [comment]
Sorry @fvictorio, I have clipped the relevant code 😁

Edited the OP
